Codebase/Aureus-Package/lib/Elements/Cards/PastelCardElement.dart

```diff
@@ -1,15 +1,37 @@
+import 'package:aureus/Elements/Info%20Hiearchy/InfoHiearchyElements.dart';
+import 'package:aureus/Elements/Misc/TitleCase.dart';
 import 'package:flutter/cupertino.dart';
-import 'package:aureus/product.dart';
 import 'package:aureus/foundation.dart';
+import 'package:flutter/material.dart';
 
 //A blank card that is a pastel version of the product color
-//Doc Link:
+//Doc Link: https://github.com/Astra-Labs/Aureus/blob/main/Documentation/Aureus-Docs/4%20-%20Elements%20(Materials)/Cards/Pastel%20Cards.md
+
+final foundation = UDSVariables();
 
 class PastelCardElement extends StatelessWidget {
+  final String cardLabel;
 
+  PastelCardElement(this.cardLabel);
   @override
   Widget build(BuildContext context) {
-    // TODO: implement build
-    throw UnimplementedError();
+    String titleCaseCardLabel = TitleCase.convertToTitleCase(cardLabel);
+    return Container(
+      height: 192,
+      width: 189,
+      decoration: BoxDecoration(
+        gradient: foundation.icyBoi2(),
+        borderRadius: BorderRadius.all(
+          Radius.circular(10),
+        ),
+        border: Border.all(
+          color: foundation.steel(),
+        ),
+      ),
+      child: Padding(
+        padding: const EdgeInsets.only(left: 13, top: 35),
+        child: BodyOneText(titleCaseCardLabel, Colors.black),
```

Comment:
depends on the Badge Card PR that has an update for the info hierarchy to accept text colour